test_runner/fixtures/neon_fixtures.py

```diff
@@ -3708,10 +3741,36 @@ def clear_shared_buffers(self, cursor: Optional[Any] = None):
 
         Might also clear LFC.
         """
+        file_cache_size_limit = 0
```

Comment:
It seems we don't use `file_cache_size_limit`, but overwrite it in both if-else blocks

Reply:
fixed